setup.py

```diff
@@ -56,6 +63,10 @@ def get_ucp_version():
 cmdclass = versioneer.get_cmdclass(cmdclass)
 
 setup(
+    # TODO: At present the ucx-py naming scheme is not dynamic and will not
+    # support different CUDA major versions if we need to build wheels. It is
+    # hardcoded in pyproject.toml, so overriding it here will not work.
+    # name="ucx-py" + os.getenv("RAPIDS_PY_WHEEL_CUDA_SUFFIX", default=""),
     ext_modules=ext_modules,
     cmdclass=cmdclass,
     version=versioneer.get_version(),
```

Comment:
The rationale for this is as for the approach we took in [dask-cuda](https://github.com/rapidsai/dask-cuda/pull/948). I copy the description:

> The GIT_DESCRIBE_TAG and VERSION_SUFFIX environment variables are used
to control the name and version of the created conda/pypi package.
They should, however, not be used to control the version of the
installed package by overriding the versioneer cmdclass since that
leaves an unmodified _version.py file in the installed package
directory. A consequence is that the version reported by
dask_cuda.__version__ is "0+unknown".

> We cannot always use the versioneer-provided cmdclass unmodified since
PEP440 specifically forbids PyPI from accepting packages that have local
version identifiers (as used by versioneer). To get around this, when setup.py
detects it is building a PyPI package (GIT_DESCRIBE_TAG is in the environment),
patch the version returned from versioneer with a PyPI-compatible one.

Specifically. If we don't use versioneer here, then the dynamic code in `__init__.py` which imports `_version` and calls `get_versions()` will report nonsense for wheel packages.